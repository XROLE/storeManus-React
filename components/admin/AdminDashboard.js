import React, { Component, Fragment } from 'react';
import Navbar from '../navbar/Navbar';

export default class AdminDashboard extends Component {
  constructor() {
    super();
  }

  render() {
    return (
      <Fragment>
        <Navbar />
        <div className="dasboardDiv">
          <div className="dasboard-content">
            <div className="dashboard-toggler-div">
              <p className="toggler">
                <i className="fa fa-bars dasboard-content-toggle-show" />
                <i className="fa fa-times close-toggle" style={{ color: 'red' }} />
              </p>
            </div>
            <div className="dasboard-content-main">
              <ul className="dasboard-content-items">
                <p className="dasboard-header"><a href="adminDashboard.html">Dashboard</a></p>
                <li className="link" id="all-products">
                  <a href="admin-all-products.html">
                    <i className="fas fa-cookie-bite" />
&nbsp; All Products
                  </a>
                </li>
                <li className="link">
                  <a href="admin-available-products.html">
                    <i className="fab fa-accessible-icon" />
&nbsp; Available Products
                  </a>
                </li>
                <li className="link">
                  <a href="admin-finished-products.html">
                    <i className="fas fa-thermometer-empty" />
&nbsp; Finished Products
                  </a>
                </li>
                <li className="link">
                  <a href="admin-add-products.html">
                    <i className="far fa-plus-square" />
&nbsp; Add Product
                  </a>
                </li>
              </ul>
              <ul className="dasboard-content-items">
                <p className="dasboard-header">Attendants</p>
                <li className="link">
                  <a href="admin-all-attendants.html">
                    <i className="fas fa-users" />
&nbsp; All Attendants
                  </a>
                </li>
                <li className="link">
                  <a href="admin-add-attendant.html">
                    <i className="far fa-plus-square" />
&nbsp; Add Attendant
                  </a>
                </li>
              </ul>
            </div>
          </div>
          <div className="test">
            <div className="avatar-div-container">
              <div className="avater-div">
                <img src="../../src/img/avatar.jpg" alt="Avatar" className="Avatar" />
                <p className="avatar-label">welcome: Xrole</p>
              </div>
              <div className="filter-div">
                <a href="admin-sales-viewer.html">Filter Sales By: </a>
&nbsp;
                <select name="" id="">
                  <option value="">-Select All-</option>
                  <option value="">Attendats</option>
                  <option value="">Categories</option>
                  <option value="">Date</option>
                </select>
&nbsp;
                <select name="" id="">
                  <option value="">---- Select All ----</option>
                  <option value="Cythia Paul">Cythia Paul</option>
                  <option value="Blessing Uche">Blessing Uche</option>
                  <option value="Silas Nwokeocha">Silas Nwokeocha</option>
                  <option value="Idowu Samson">Idowu Samson</option>
                  <option value="Mathew Jade">Mathew Jade</option>
                  <option value="Will Smith">Will Smith</option>
                  <option value="Cindy Eze">Cindy Eze</option>
                  <option value="Simon Peter">Simon Peter</option>
                </select>
                        &nbsp;
                <button className="search-sales-by-attendant-buttton link" type="submit">
                  <i className="fas fa-search" />
                  <span className="search-text"> &nbsp; Search</span>
                </button>
              </div>
            </div>
            <div className="dasboard-display-div" style={{ width: '90%' }} id="testing">
              <img src="../../src/img/shopping-cart.jpg" id="shopping-cart" alt="" style={{ width: '50%', marginTop: '5%' }} />
            </div>
          </div>
        </div>
      </Fragment>
    );
  }
}
