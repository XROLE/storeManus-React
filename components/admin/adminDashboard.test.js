import React from 'react';
import { shallow } from 'enzyme';
import AdminDashboard from './AdminDashboard';

describe('Admin Dashboard', () => {
  it('Should render just the Admin Dashboard component', () => {
    const component = shallow(
      <AdminDashboard />,
    );
    expect(component.exists()).toBe(true);
    expect(component).toMatchSnapshot();
  });
});
