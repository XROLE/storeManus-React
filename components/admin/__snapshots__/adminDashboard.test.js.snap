// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin Dashboard Should render just the Admin Dashboard component 1`] = `
<Fragment>
  <Navbar />
  <div
    className="dasboardDiv"
  >
    <div
      className="dasboard-content"
    >
      <div
        className="dashboard-toggler-div"
      >
        <p
          className="toggler"
        >
          <i
            className="fa fa-bars dasboard-content-toggle-show"
          />
          <i
            className="fa fa-times close-toggle"
            style={
              Object {
                "color": "red",
              }
            }
          />
        </p>
      </div>
      <div
        className="dasboard-content-main"
      >
        <ul
          className="dasboard-content-items"
        >
          <p
            className="dasboard-header"
          >
            <Link
              replace={false}
              to="/adminDashboard"
            >
              Dashboard
            </Link>
          </p>
          <li
            className="link"
            id="all-products"
          >
            <a
              href="admin-all-products.html"
            >
              <i
                className="fas fa-cookie-bite"
              />
                All Products
            </a>
          </li>
          <li
            className="link"
          >
            <a
              href="admin-available-products.html"
            >
              <i
                className="fab fa-accessible-icon"
              />
                Available Products
            </a>
          </li>
          <li
            className="link"
          >
            <a
              href="admin-finished-products.html"
            >
              <i
                className="fas fa-thermometer-empty"
              />
                Finished Products
            </a>
          </li>
          <li
            className="link"
          >
            <Link
              replace={false}
              to="admin/add/product"
            >
              <i
                className="far fa-plus-square"
              />
                Add Product
            </Link>
          </li>
        </ul>
        <ul
          className="dasboard-content-items"
        >
          <p
            className="dasboard-header"
          >
            Attendants
          </p>
          <li
            className="link"
          >
            <a
              href="admin-all-attendants.html"
            >
              <i
                className="fas fa-users"
              />
                All Attendants
            </a>
          </li>
          <li
            className="link"
          >
            <Link
              replace={false}
              to="admin/add/attendant"
            >
              <i
                className="far fa-plus-square"
              />
                Add Attendant
            </Link>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="test"
    >
      <div
        className="avatar-div-container"
      >
        <div
          className="avater-div"
        >
          <img
            alt="Avatar"
            className="Avatar"
            src="../../src/img/avatar.jpg"
          />
          <p
            className="avatar-label"
          >
            welcome: Xrole
          </p>
        </div>
        <div
          className="filter-div"
        >
          <a
            href="admin-sales-viewer.html"
          >
            Filter Sales By: 
          </a>
           
          <select
            id=""
            name=""
          >
            <option
              value=""
            >
              -Select All-
            </option>
            <option
              value=""
            >
              Attendats
            </option>
            <option
              value=""
            >
              Categories
            </option>
            <option
              value=""
            >
              Date
            </option>
          </select>
           
          <select
            id=""
            name=""
          >
            <option
              value=""
            >
              ---- Select All ----
            </option>
            <option
              value="Cythia Paul"
            >
              Cythia Paul
            </option>
            <option
              value="Blessing Uche"
            >
              Blessing Uche
            </option>
            <option
              value="Silas Nwokeocha"
            >
              Silas Nwokeocha
            </option>
            <option
              value="Idowu Samson"
            >
              Idowu Samson
            </option>
            <option
              value="Mathew Jade"
            >
              Mathew Jade
            </option>
            <option
              value="Will Smith"
            >
              Will Smith
            </option>
            <option
              value="Cindy Eze"
            >
              Cindy Eze
            </option>
            <option
              value="Simon Peter"
            >
              Simon Peter
            </option>
          </select>
           
          <button
            className="search-sales-by-attendant-buttton link"
            type="submit"
          >
            <i
              className="fas fa-search"
            />
            <span
              className="search-text"
            >
                 Search
            </span>
          </button>
        </div>
      </div>
      <div
        className="dasboard-display-div"
        id="testing"
        style={
          Object {
            "width": "90%",
          }
        }
      >
        <img
          alt=""
          id="shopping-cart"
          src="../../src/img/shopping-cart.jpg"
          style={
            Object {
              "marginTop": "5%",
              "width": "50%",
            }
          }
        />
      </div>
    </div>
  </div>
</Fragment>
`;
